config/config.go

```diff
@@ -721,6 +721,10 @@ func New(v *viper.Viper, bidderInfos BidderInfos, normalizeBidderName func(strin
 		return nil, err
 	}
 
+	if err := UnpackDSADefault(c.AccountDefaults.Privacy.DSA); err != nil {
+		return nil, err
```

Comment:
The error returned does not point to dsa. I suggest formatting this to be more descriptive, something along the lines of `"Invalid default account DSA %v", err`.